apps/nova/src/pages/UpcomingLessons.tsx

```diff
@@ -21,10 +55,35 @@ const UpcomingLessons: React.FC = () => {
     canceled: true,
   });
 
+  // useMutation hooks
+  const { mutate: createRating } = useCreateRatingTutor({
+    onSuccess: () => setRateDialogInfo((prev) => ({ ...prev, enabled: false })),
+    onError: () => console.log("createRating failed!"),
```

Comment:
The one error should look like this [one](https://github.com/litespace-org/litespace/blob/14648c2e5d068f53d47912de3730bf2dfe2bafb7/apps/nova/src/components/TutorProfile/RateTutor.tsx#L24-L30) (show an errored-toast message)